### ISSUE TITLE
Broken map links.

### ISSUE THREAD

**Issue by AnthonyHJ** [ChatTheatre/orchil#25]
When requesting the full map, the server sends SKOOT 4 with a valid URL, but Orchil is prepending "http://marrach.skotos.net/Theatre/Art.sam?url=" (no longer there) to the window. 

I'm pretty sure this should be in a config file somewhere, since it seems oddly specific, but it's actually hard-coded into the "popupArtWin" function.

**Comment by AnthonyHJ** [comment]
Okay... It seems to be hard-coded in CM, but not in the version on GitHub...

**Comment by noahgibbs** [comment]
Yeah, ChatTheatre's versions of Orchil and SkotOS both pretty clearly expect SkotOS to fully generate its own URLs. That's not perfect, but it's what they currently do.
